apps/base-docs/tutorials/docs/1_event-gate-nouns.md

```diff
@@ -0,0 +1,429 @@
+
+:::
+
+### Artwork
+
+Add a description for your DAO. You must put something in this field.
+
+For the art, click the `Download demo folder` link and review the assets downloaded if you're not familiar with how these types of collections assemble NFT art out of random properties.
+
+When you're done, simply upload the folder you downloaded. Use the `Generate Random Preview` button a few times to see the variants of NFTs this protocol can create, then click `Continue`.
+
+### Deploying the Contracts
+
+Review the linked documents for the ToS and Protocol Rewards, the check the boxes and click `Deploy Contracts (1 of 2)`.
+
+Approve the transaction. **Remember, this is using real funds!**
+
+Click `Deploy Contracts (2 of 2)` and approve the transaction. After it processes, you should be taken to the auction page for your DAO. Click `Start Auction` and approve the transaction.
+
+If you chose to distribute tokens to founding addresses, those will be airdropped to those addresses. Otherwise, use the auction to purchase NFTs with a few different addresses.
+
+![Auction](../../assets/images/build-with-nouns/auction.png)
+
+### Finding the Contracts
+
+Near the bottom of the page, click the `Contracts` tab. You'll need the `NFT` contract address for the app.
+
+## Building the App
+
+The app is designed to be used by the person at the door to a private event for DAO members. To gain entry, members must prove their ownership of at least 1 NFT from the DAO.
+
+The app will work by using a QR code scanner to read the member's "pay me" QR code, look up their NFT ownership status, then share that with the person using the app to scan entrants.
+
+### Setting up the Template
+
+Start by using [An Onchain App in 100 Components] as a template.
+
+Clone your copy of the repo, install dependencies, and check that it works.
+
+Open `src/app/page.tsx`. Clean out the demo content and set up for your app:
+
+```tsx
+'use client';
+
+export default function Page() {
+  return (
+    <div className="flex w-96 flex-col md:w-[600px]">
+      <section className="mb-6 flex w-full flex-col border-b border-sky-800 pb-6">
+        <aside className="mb-6 flex">
+          <h2 className="text-2xl">Welcome to our Event!</h2>
+        </aside>
+        <main className="flex flex-col space-x-4">
+          <p className="text-body mb-4 text-white">
+            Please scan the QR code for your wallet address that holds the NFT.
+          </p>
+        </main>
+      </section>
+    </div>
+  );
+}
+```
+
+### Reading QR Codes
+
+You'll need a library to read QR codes. A lightweight option is [Html5-QRCode].
+
+Add it with:
+
+```bash
+bun add html5-qrcode
+```
+
+In `page.tsx`, add the state variables needed by the scanner, as outlined in the [Html5-QRCode Docs], and the variables you'll need to show whether or not to let the person into the event:
+
```

Comment:
I would prompt the reader to import the QR scanner. 

`import { Html5Qrcode } from 'html5-qrcode';`